home.html

```diff
@@ -0,0 +1,34 @@
+<html>
```

Comment:
A `doctype` is needed at the top -- `<!DOCTYPE html>`. It works without it, but it's needed to be correct.